src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/ClientCreds/ClientKey.cs

```diff
@@ -95,6 +114,7 @@ public ClientKey(ClientAssertion clientAssertion)
 
         public bool HasCredential { get; private set; }
 
+        public bool SendX5c { get; private set; }
```

Comment:
>public bool SendX5c { get; private set; } [](start = 8, length = 41)

though instead of setting this in the constructors: public string SendX5c { get; set; } = false; ( this is btw. the default value of a bool in C#)